src/virtualship/instruments/xbt.py

```diff
@@ -0,0 +1,137 @@
+"""XBT instrument."""
+
+from dataclasses import dataclass
+from datetime import timedelta
+from pathlib import Path
+
+import numpy as np
+from parcels import FieldSet, JITParticle, ParticleSet, Variable
+
+from ..spacetime import Spacetime
+
+
+@dataclass
+class XBT:
+    """Configuration for a single XBT."""
+
+    spacetime: Spacetime
+    min_depth: float
+    max_depth: float
+    fall_speed: float
+    deceleration_coefficient: float
+
+
+_XBTParticle = JITParticle.add_variables(
+    [
+        Variable("temperature", dtype=np.float32, initial=np.nan),
+        Variable("max_depth", dtype=np.float32),
+        Variable("min_depth", dtype=np.float32),
+        Variable("fall_speed", dtype=np.float32),
+        Variable("deceleration_coefficient", dtype=np.float32),
+    ]
+)
+
+
+def _sample_temperature(particle, fieldset, time):
+    particle.temperature = fieldset.T[time, particle.depth, particle.lat, particle.lon]
+
+
+def _xbt_cast(particle, fieldset, time):
+    particle_ddepth = -particle.fall_speed * particle.dt
+
+    # update the fall speed from the quadractic fall-rate equation
+    # check https://doi.org/10.5194/os-7-231-2011
+    particle.fall_speed = (
+        particle.fall_speed - 2 * particle.deceleration_coefficient * particle.dt
+    )
+
+    # delete particle when it reaches the maximum depth
+    if particle.depth + particle_ddepth < particle.max_depth:
+        particle.delete()
```

Comment:
@erikvansebille 
Now, I am updating the fall speed with the `deceleration_coefficient` and deleting the particle if it reaches the maximum depth. We can also add the papers and the description to the documentation.
I follow the example from `ctd.py`, but I don't see `particle.depth` being updated. Is this correct? How does that work?

Reply:
> @erikvansebille Now, I am updating the fall speed with the `deceleration_coefficient` and deleting the particle if it reaches the maximum depth. 

OK

>We can also add the papers and the description to the documentation. 

Yes, good idea to add the references!

> I follow the example from `ctd.py`, but I don't see `particle.depth` being updated. Is this correct? How does that work?

The updating happens through the `particle_ddepth` variable, see the [Kernel loop tutorial](https://docs.oceanparcels.org/en/latest/examples/tutorial_kernelloop.html)

Reply:
Should I "particle.depth = particle_ddepth" ?

Reply:
> Should I particle.depth = particle_ddepth ?

No, that's done automatically in the backend; you shouldn't change `particle.depth` at all.